components/mailer.rst

```diff
@@ -16,8 +16,129 @@ Installation
 
 .. include:: /components/require_autoload.rst.inc
 
+
+Introduction
+------------
+
+
 Usage
 -----
 
-We're currently working on the documentation of this component that was just
-added to Symfony. We'll publish it in a few days.
+    use Symfony\Component\Mailer\Mailer;
+    use Symfony\Component\Mailer\Transport\Smtp\SmtpTransport;
+
+    $transport = new SmtpTransport('localhost');
+    $mailer = new Mailer($transport);
+    $mailer->send($email);
+
+Refer :doc:`Mime component </components/mime>` how to create `$email` object.
+
+
+Transport
+---------
+
+By default, the only transport available in the mailer component is Smtp.
+Below is the list of other popular providers with built in support.
+- Amazon SES  : symfony/amazon-mailer
+- Google Gmail : symfony/google-mailer
+- Mandrill : symfony/mailchimp-mailer
+- Mailgun : symfony/mailgun-mailer
+- Postmark : symfony/postmark-mailer
+- Sendgrid : symfony/sendgrid-mailer
+
+For example to use google's gmail as a transport you need to install symfony/google-mailer.
+
+.. code-block:: terminal
+
+    $ composer require symfony/google-mailer
+
+.. include:: /components/require_autoload.rst.inc
+
+
+    use Symfony\Component\Mailer\Bridge\Google\Smtp\GmailTransport;
```

Comment:
Above this, add a:

```rst
.. code-block:: php
```

Reply:
cool. Didn't added for looking at mime, there was none. Will add.